src/NServiceBus.AcceptanceTests/Basic/When_multiple_mappings_exists.cs

```diff
@@ -55,8 +54,7 @@ public class MyMessageHandler : IHandleMessages<MyBaseCommand>
                 public Task Handle(MyBaseCommand message)
```

Comment:
Not related to this PR - should `IHandleMessages<T>.Handle(T)` be suffixed with Async?